lib/Twig/Extension/Core.php

```diff
@@ -991,7 +990,7 @@ function twig_escape_filter(Twig_Environment $env, $string, $strategy = 'html',
                 $string = twig_convert_encoding($string, 'UTF-8', $charset);
             }
 
-            if (0 == strlen($string) ? false : (1 == preg_match('/^./su', $string) ? false : true)) {
+            if ('' === $string ? false : (1 === preg_match('/^./su', $string) ? false : true)) {
```

Comment:
like this https://github.com/twigphp/Twig/pull/2256/commits/15a7677aa271be4540860e48877c4f79e153c2ba ?